### DISCUSSION TITLE
Community detection with pyg ?

### DISCUSSION THREAD

**Discussion by jdg-stat** [pyg-team/pytorch_geometric#8960] [Q&A]
I am currently looking for some tutorial or starter code on doing community detection using pytorch geometric but there doesn't seem to be much material available. If someone can point me to some tutorial or provide a sample code that would be awesome.

**Comment by rusty1s** [comment]
Yeah, PyG misses an example for this. I could only point you to some research papers right now, *e.g.*, [this one](https://arxiv.org/abs/2401.02542).
